src/aspire/source/simulation.py

```diff
@@ -13,44 +13,43 @@
 
 class Simulation(ImageSource):
     def __init__(self, L=8, n=1024, states=None, filters=None, offsets=None, amplitudes=None, dtype='single', C=2,
-                 rots=None):
+                 angles=None):
         """
         A Cryo-EM simulation
         Other than the base class attributes, it has:
 
         :param C: The no. of distinct volumes
-        :param rots: A 3-by-3-by-n array of rotation matrices corresponding to viewing directions
+        :param angles: A 3-by-n array of rotation angles
         """
+        super().__init__(L=L, n=n, dtype=dtype)
 
         offsets = offsets or L / 16 * randn(2, n, seed=0).T
         if amplitudes is None:
             min_, max_ = 2./3, 3./2
             amplitudes = min_ + rand(n, seed=0) * (max_ - min_)
         states = states or randi(C, n, seed=0)
-        rots = rots or angles_to_rots(self._uniform_random_angles(n, seed=0))
-
-        super().__init__(
-            L=L,
-            n=n,
-            states=states,
-            filters=filters,
-            offsets=offsets,
-            amplitudes=amplitudes,
-            rots=rots,
-            dtype=dtype
-        )
-
+        angles = angles or self._uniform_random_angles(n, seed=0)
```

Comment:
We should allow the user to specify a seed for the simulation. Sometimes we'll want to run the same simulation multiple times, but with some randomness in each trial.

Reply:
Fixed in cfc6e40895f49fe43a864c94ca75964e5d90a218